src/JsonBackupProfile.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace MWStake\MediaWiki\CliAdm;
+
+class JsonBackupProfile implements IBackupProfile {
+
+	/**
+	 * @var array
+	 */
+	protected $data = [
+		'fs-options' => [
+			'skip-image-paths' => [
+				'thumb',
+				'temp',
+				'cache'
+			]
+		],
+		'db-options' => [
+			'skip-tables' => [
+				'objectcache',
+				'l10n_cache',
+				'bs_whoisonline',
+				'smw_.*?'
+			]
+		]
+	];
+
+	/**
+	 * @param string $jsonFilePathname
+	 */
+	public function __construct( string $jsonFilePathname ) {
+		$contents = file_get_contents( $jsonFilePathname );
+		$filedata = json_decode( $contents, true );
+
+		$this->data = array_merge( $this->data, $filedata );
```

Comment:
I understand that this will allow the user to specify custom `fs-options` but not having to repeat the default `db-options`. Am I right?

Reply:
Yes, you're completely right.